ordering/__init__.py

```diff
@@ -103,6 +104,14 @@ def assert_new_item(self, item: T) -> None:
         if item in self:
             raise KeyError("Ordering {} already contains {}".format(self, item))
 
+    @classmethod
+    def assert_unique_items(cls, items: Collection[_T]) -> None:
+        if len(items) > len(set(items)):
+            d: Dict[_T, bool] = {}
+            duplicates = [i for i in items if i in d or d.setdefault(i, False)]
+            raise ValueError('attempt to create {} containing duplicate items: {}'
```

Comment:
I use [this](https://stackoverflow.com/a/56190) convention for single/double quotes.